### PR TITLE
Clean up code style warnings

### LINKED ISSUE #1387: MobiFlightCache.cs has a bunch of code correctness warnings
The work in #1364 added a bunch of new code that has code correctness warnings in Visual Studio. None of them affect the functionality but we should clean them up.

### PR DESCRIPTION
**Pull request by neilenns**
Fixes #1387 

See comments on the invidial things in the PR for details. The main file to look at is `mobiflightcache.cs`. The only reason the other files got touched (with one noted exception in the PR comments) is from functions being renamed.

### CODE REVIEW

File: MobiFlight/MobiFlightCache.cs [hunk 1]
```diff
@@ -42,23 +42,21 @@ public class MobiFlightCache : MobiFlightCacheInterface, ModuleCacheInterface
         public event EventHandler LookupFinished;
 
 
+#pragma warning disable IDE0044 // Add readonly modifier.
         private volatile List<MobiFlightModuleInfo> AvailableComModules = new List<MobiFlightModuleInfo>();
```
Comment (neilenns) [inline MobiFlight/MobiFlightCache.cs:46]: VS really wants this to be marked as readonly, but you can't do readonly and volatile at the same time, so ignore the warning.

File: MobiFlight/MobiFlightCache.cs [hunk 2]
```diff
@@ -42,23 +42,21 @@ public class MobiFlightCache : MobiFlightCacheInterface, ModuleCacheInterface
...
 
 
+#pragma warning disable IDE0044 // Add readonly modifier.
         private volatile List<MobiFlightModuleInfo> AvailableComModules = new List<MobiFlightModuleInfo>();
+#pragma warning restore IDE0044 // Add readonly modifier
         Boolean isFirstTimeLookup = true;
 
-        private bool _lookingUpModules = false;
-
-        DateTime servoTime = DateTime.Now;
```
Comment (neilenns) [inline MobiFlight/MobiFlightCache.cs:50]: These were both unused.

File: MobiFlight/MobiFlightCache.cs [hunk 3]
```diff
@@ -535,26 +532,15 @@ public void setDisplayBrightness(string serial, string address, byte connector,
...
 
-        private string GetValueForReference(string reference, List<ConfigRefValue> referenceList)
-        {
-            if (referenceList == null)
-            {
-                return null;
-            }
-            var found = referenceList.Find(x => x.ConfigRef.Ref.Equals(reference));
-            return found?.Value;
-        }
```
Comment (neilenns) [inline MobiFlight/MobiFlightCache.cs:546]: This entire function is never used anywhere.

File: MobiFlight/MobiFlightCache.cs [hunk 4]
```diff
@@ -818,7 +803,7 @@ public List<String> GetMobiFlightVariableNames()
             return result;
         }
 
-        internal void Set(string serial, OutputConfig.CustomDevice deviceConfig, string value, List<ConfigRefValue> configRefValues)
```
Comment (neilenns) [inline MobiFlight/MobiFlightCache.cs:821]: The `configRefValues` parameter isn't used.

File: MobiFlight/ExecutionManager.cs [hunk 5]
```diff
@@ -864,7 +864,7 @@ private string ExecuteDisplay(string value, OutputConfigItem cfg)
                         break;
 
                     case OutputConfig.CustomDevice.Type:
-                        mobiFlightCache.Set(serial, cfg.CustomDevice, value, GetRefs(cfg.ConfigRefs));
```
Comment (neilenns) [inline MobiFlight/ExecutionManager.cs:867]: Despite being passed in, `cfg.ConfigRefs` is never used by the method.

File: MobiFlight/ExecutionManager.cs [hunk 6]
```diff
@@ -787,7 +787,7 @@ private string ExecuteDisplay(string value, OutputConfigItem cfg)
                         {
                             string refValue = FindValueForRef(cfg.LedModule.DisplayLedBrightnessReference);
 
-                            mobiFlightCache.setDisplayBrightness(
+                            mobiFlightCache.SetDisplayBrightness(
```
Comment (DocMoebiuz) [inline MobiFlight/ExecutionManager.cs:790]: This kind of change (and subsequent) are not warnings. These are messages.

File: MobiFlight/ExecutionManager.cs (hunk 5 above)
Comment (DocMoebiuz) [reply MobiFlight/ExecutionManager.cs:867]: I don't remember if that was some idea which we had at some point.

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]

File: MobiFlight/MobiFlightCache.cs [hunk 7]
```diff
@@ -785,11 +769,12 @@ public List<String> GetMobiFlightVariableNames()
 
         public Dictionary<String, int> GetStatistics()
         {
-            Dictionary<String, int> result = new Dictionary<string, int>();
-
-            result["Modules.Count"] = Modules.Values.Count();
+            Dictionary<String, int> result = new Dictionary<string, int>
```
[the remark this answers is not in the document]
Comment (neilenns) [reply MobiFlight/MobiFlightCache.cs:772]: I only touched things that Visual Studio put a code correctness message out for. It didn't have one for "use var" so I didn't.

File: MobiFlight/ExecutionManager.cs (hunk 6 above)
Comment (neilenns) [reply MobiFlight/ExecutionManager.cs:790]: Yeah, that's what I meant. They're code style warnings but report as messages in the errors list.

File: MobiFlight/MobiFlightCache.cs (hunk 7 above)
[the remark this answers is not in the document]
Comment (DocMoebiuz) [reply MobiFlight/MobiFlightCache.cs:772]: Just change it, makes it consistent.

[the remark this answers is not in the document]
Comment (neilenns) [reply MobiFlight/MobiFlightCache.cs:772]: You'd think Visual Studio's auto-fix would use `var` instead 😅 But noooooooo. I fixed this.

**Review by DocMoebiuz — APPROVED** [review]
THANKS!

**Pull request merged**